microservices/ItemService/Controllers/BooksController.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ItemService.Models;
+
+namespace ItemService.Controllers
+{
+    [Route("store")]
+    [ApiController]
+    public class BooksController(BookContext context) : ControllerBase
+    {
+        private readonly BookContext _context = context;
+
+        // GET: store/items
+        [HttpGet("items")]
+        public async Task<ActionResult<IEnumerable<Book>>> GetItems()
+        {
+            return await _context.Books.ToListAsync();
+        }
+
+        // GET: store/items/5
+        [HttpGet("items/{id}")]
+        public async Task<ActionResult<Book>> GetItem(ulong id)
+        {
+            var item = await _context.Books.FindAsync(id);
```

Comment:
item should be renamed to book!!!!!!!!

Reply:
Fixed in commit [2f89cb5](https://github.com/587900/dat251-book-store/pull/2/commits/2f89cb5bc5997c9b1e0f52e01f1a69fd327a8e87)